src/frontend/js/components/SaleTunnel/index.full-process.spec.tsx

```diff
@@ -273,7 +273,7 @@ describe('SaleTunnel', () => {
     /**
      * Make sure the checkbox to waive withdrawal right is displayed
      */
-    const $waiveCheckbox = screen.getByLabelText('I waive my right of withdrawal');
+    const $waiveCheckbox = screen.getByLabelText('By checking this box:', { exact: false });
```

Comment:
Actually, that's because the text label embedded also text items. So the exact match is `By checking this box: I acknowledge that I have expressly requested to begin the training before the expiration date of the withdrawal period. I expressly waive my right of withdrawal in order to begin the training before the expiration of the withdrawal period.`

But with our comment, I rewrite this part to retrieve the checkbox element without any depends to the label text.

```
const $waiveCheckbox = within(screen.getByTestId('withdraw-right-checkbox')).getByRole('checkbox');
```